assets/install_controller.sh

```diff
@@ -51,6 +51,28 @@ load_existing_nvm() {
 	set -e
 }
 
+controller_service() {
+    USE_SYSTEMD=`grep -m1 -c systemd /proc/1/comm`
+    USE_INITCTL=`which /sbin/initctl | wc -l`
```

Comment:
Do you need to specify the exact path ? can't we do `which initctl` ? (What if initctl had been installed somewhere else)